core/src/components/header/header.ionic.scss

```
@@ -0,0 +1,13 @@
+@use "./header.common";
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+
+// Ionic Header
+// --------------------------------------------------
+
+ion-header {
+  --box-shadow: #{globals.$ionic-elevation-200};
+
+  box-shadow: var(--box-shadow);
```

Are we expecting the box shadow to change? If not, then let's follow the structure that is established for `md`:

```suggestion
  .header-ionic {
     box-shadow: #{globals.$ionic-elevation-200};
  }
```